osu.Game.Rulesets.Osu/Edit/Blueprints/Sliders/Components/SliderBodyPiece.cs

```diff
@@ -61,10 +61,20 @@ public override void UpdateFrom(Slider hitObject)
                 body.SetVertices(vertices);
             }
 
-            Size = body.Size;
             OriginPosition = body.PathOffset;
         }
 
+        public override Vector2 Size
+        {
+            get
+            {
+                if (base.Size != body.Size)
+                    Size = body.Size;
+                return base.Size;
+            }
```

Comment:
A getter _cannot_ mutate data. This is how you make future developers go insane. That way lies madness.

Reply:
... okay thanks, I... don't know why I forgot that, honestly. I did think of some ways to handle things that could come of it (namely, recursion and issues with inheritance), but never stopped to ask myself that maybe this is a bad idea in the first place.

I did want to override the getter to just reference `body.Size` instead, however this seems like a bad idea with classes derived from Drawable, as Height and Width (amongst a few other methods iirc) would become desynced with Size.